vdr/src/contracts/did/types/did.rs

```diff
@@ -1,8 +1,27 @@
 use crate::{types::ContractOutput, ContractParam, VdrError, VdrResult};
+use once_cell::sync::Lazy;
+use regex_lite::Regex;
 use serde_derive::{Deserialize, Serialize};
 
 pub const DID_PREFIX: &str = "did";
 
+const DID_SYNTAX: &str = r"did:(?:indybesu|ethr):(?:[a-zA-Z0-9]+:)*0x[a-fA-F0-9]{40}";
```

Comment:
```suggestion
const DID_SYNTAX: &str = r"did:(?:indy:besu|ethr):(?:[a-zA-Z0-9]+:)*0x[a-fA-F0-9]{40}";
```
We need to update this in the next PR according to Indy DID method
https://github.com/hyperledger/indy-did-method/pull/86